app/controllers/api/v1/trends/tags_controller.rb

```diff
@@ -5,7 +5,7 @@ class Api::V1::Trends::TagsController < Api::BaseController
 
   after_action :insert_pagination_headers
 
-  DEFAULT_TAGS_LIMIT = 10
+  TAGS_LIMIT = (ENV['MAX_TRENDING_TAGS'] || 10).to_i
```

Comment:
Still calling it `DEFAULT_TAGS_LIMIT` would reduce the amount of changes from upstream, and thus risks of merge conflicts.

Reply:
Understood! I'll keep that in mind.